WordPress/Classes/WordPress.xcdatamodeld/WordPress 155.xcdatamodel/contents

```diff
@@ -182,6 +185,7 @@
         <relationship name="postTypes" optional="YES" toMany="YES" deletionRule="Cascade" destinationEntity="PostType" inverseName="blog" inverseEntity="PostType" syncable="YES"/>
         <relationship name="publicizeInfo" optional="YES" maxCount="1" deletionRule="Cascade" destinationEntity="PublicizeInfo" inverseName="blog" inverseEntity="PublicizeInfo"/>
         <relationship name="quickStartTours" optional="YES" toMany="YES" deletionRule="Cascade" destinationEntity="QuickStartTourState" inverseName="blog" inverseEntity="QuickStartTourState" syncable="YES"/>
+        <relationship name="rawBlockEditorSettings" optional="YES" maxCount="1" deletionRule="Cascade" destinationEntity="BlobEntity" syncable="YES"/>
```

Comment:
Note the "cascade" deletion rule, which is what you need.

Reply:
Thank you for noting this.